### PR TITLE
Aggregate fns

### PR DESCRIPTION
**Pull request by Thoth1111**
In this branch, I :
✅ Used database transactions.
✅ Modified and deleted data in SQL.
✅ Prepared complex queries that answered analytical questions using aggregate functions.

The following are screenshots of the operations in order of execution:

Screenshot (16)
Screenshot (17)
Screenshot (18)
Screenshot (19)
Screenshot (20)
Screenshot (21)
Screenshot (22)
Screenshot (23)
Screenshot (24)
Screenshot (25)
Screenshot (26)
Screenshot (27)
Screenshot (28)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -3,3 +3,53 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALU
...
+VALUES (
+    'Blossom', '1998-10-13', 3, true, 17 
+);
+
+INSERT INTO animals (
+	name,date_of_birth,escape_attempts,neutered,weight_kg
+)
+VALUES (
+    'Ditto', '2022-05-14', 4, true, 22 
+);
```
Comment (ShadyShawkat) [inline data.sql]: - [x] Please make sure that the entered data is the same as the data given in the exercise. Some of the given `weight_kg` values should be minus (`-`). This is intended and not a mistake as they are going to be fixed later with queries.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Thoth1111, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Thoth1111) [reply data.sql]: Thank you @ShadyShawkat. I didn't notice the values were negative.

**Comment by Thoth1111** [comment]
Here is what I've done to address the changes requested by the last reviewer:
✅ I made updates to the values in the database to negate them.
Screenshot (29)

✅ I repeated this query from the project requirements: _"Update all animals' weights that are negative to be their weight multiplied by -1."_
Screenshot (30)

✅I modified the data.sql file to match the negative values from the requirements.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**